geospaas/base_viewer/forms.py

```diff
@@ -0,0 +1,60 @@
+from django.contrib.gis import forms
+from django.utils import timezone
+from django.db.models import Q
+
+from leaflet.forms.widgets import LeafletWidget
+from geospaas.catalog.models import Dataset as CatalogDataset
```

Comment:
If you don't have another `Dataset` class in this file, you don't need to give it an alias